thrift/tchannel.thrift

```diff
@@ -0,0 +1,8 @@
+struct ClientLibrary {
```

Comment:
Ok my only concern then is that we (humans) would basically have to address workers individually to discover a service's version info?